README.md

```diff
@@ -482,6 +485,31 @@ redistribute => [['direct'],
                  ['ospf 3',  'rm_ospf'],
                  ['rip 4']]
 ```
+
+##### `route target both auto`
+Enable/Disable route target both auto only for IBGP. Valid values are true, false, or 'default'.
+
+##### `route target both auto evpn`
+Enable/Disable route target both auto evpn only for IBGP. Valid values are true, false, or 'default'.
```

Comment:
##### `route_target_both_auto_evpn`
(iBGP only, EVPN only) Enable/Disable the EVPN route-target 'auto' setting for both import and export target communities. Valid values are true, false, or 'default'.
```